src/system.c

```diff
@@ -35,9 +35,9 @@
 #include "gen_bpf.h"
 #include "helper.h"
 
-/* NOTE: the seccomp syscall whitelist is currently disabled for testing
+/* NOTE: the seccomp syscall allowlist is currently disabled for testing
  *       purposes, but unless we can verify all of the supported ABIs before
- *       our next release we may have to enable the whitelist */
+ *       our next release we may have to enable the allowlist */
 #define SYSCALL_WHITELIST_ENABLE	0
```

Comment:
Do you want to change the name of this #define as well?

Reply:
/me smacks forehead

Yes, that needs to be changed as well.  Can you tell I didn't pass the '-i' argument to `grep`? :)

Reply:
:)